Xamarin.Forms.Core/Items/SelectableItemsView.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Windows.Input;
+
+namespace Xamarin.Forms
+{
+	public class SelectableItemsView : ItemsView
+	{
+		public static readonly BindableProperty SelectionModeProperty =
+			BindableProperty.Create(nameof(SelectionMode), typeof(SelectionMode), typeof(SelectableItemsView),
+				SelectionMode.None);
+
+		public static readonly BindableProperty SelectedItemProperty =
+			BindableProperty.Create(nameof(SelectedItem), typeof(object), typeof(SelectableItemsView),
+				propertyChanged: SelectedItemPropertyChanged);
+
+		public static readonly BindableProperty SelectionChangedCommandProperty =
+			BindableProperty.Create(nameof(SelectionChangedCommand), typeof(ICommand), typeof(SelectableItemsView));
+
+		public static readonly BindableProperty SelectionChangedCommandParameterProperty =
+			BindableProperty.Create(nameof(SelectionChangedCommandParameter), typeof(object),
+				typeof(SelectableItemsView));
+
+		public object SelectedItem
+		{
+			get => GetValue(SelectedItemProperty);
+			set => SetValue(SelectedItemProperty, value);
+		}
```

Comment:
for a braces lover, that's a surprisingly low number of lines and braces for a single property 🤪